### PR TITLE
ipa_tests: test ssh keys login

### PR DESCRIPTION
**Pull request by Rezney**
Integration test for:

https://pagure.io/SSSD/sssd/issue/3747

### CODE REVIEW

**Comment by rcritten** [comment]
ipatests/test_integration/test_commands.py:206: [E1101(no-member), TestIPACommand.test_ssh_key_connection] Module 'tempfile' has no 'TemporaryDirectory' member)

File: ipatests/test_integration/test_commands.py [hunk 1]
```diff
@@ -187,3 +194,61 @@ def test_list_help_topics(self):
...
+
+    def test_ssh_key_connection(self):
+        """
+        Integration test for https://pagure.io/SSSD/sssd/issue/3747
+        """
+
+        test_user = 'test-ssh'
+        master = self.master.hostname
+
+        with tempfile.TemporaryDirectory() as tmpdirname:
```
Comment (netoarmando) [inline ipatests/test_integration/test_commands.py]: I think you can use pytest's [tmpdir](https://docs.pytest.org/en/2.7.3/tmpdir.html), it supports Python 2.

Comment (Rezney) [reply ipatests/test_integration/test_commands.py]: @netoarmando Thanks for checking this. Yep, changed. Lets wait for the tests...

**Comment by Tiboris** [comment]
master:

* 0aeccc08524931b0e37773b4836734bc32eca7f4 ipa_tests: test ssh keys login

**Pull request closed without merging**